### PR TITLE
Added missing parameter and added default value of parameters

### LINKED ISSUE #3455: Parameter related issue on queue consumers
# Bug report
There is parameter are missing with on queue:consumers:start command

## Description
Check document https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-queue.html

### PR DESCRIPTION
**Pull request by gelanivishal**
## This PR is a:

- [X] New topic
- [X] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

 Fixes #3455

When this pull request is merged, it will update missing parameters in consumers and also added a default value of each parameter.

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/config-guide/mq/manage-mysql.html
- https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-queue.html

whatsnew
Added a missing parameter to the message queue consumer [reference documentation](https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-queue.html).

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/config-guide/mq/manage-mysql.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Edit */app/etc/env.php* file for configure cron job `consumers_runner`
 
 Use the `magento` command to start message queue consumers. You can start multiple consumers simultaneously.
 
-    ./bin/magento queue:consumers:start [--max-messages=<value>] [--batch-size=<value>] [--pid-file-path=<value>] <consumer_name>
+    ./bin/magento queue:consumers:start <consumer_name> [--max-messages=<value>] [--batch-size=<value>] [--pid-file-path=<value>] [--area-code=<value>]
```
Comment (jeff-matthews) [inline guides/v2.2/config-guide/mq/manage-mysql.md]: Does this not also apply to 2.1 and 2.3?

Comment (gelanivishal) [reply guides/v2.2/config-guide/mq/manage-mysql.md]: I have updated for v2.1. Check it.

File: _includes/config/message-queue-consumers.md [hunk 2]
```diff
@@ -21,17 +21,18 @@ async.operations.all
 To start message queue consumers:
 
 ```bash
-bin/magento queue:consumers:start [--max-messages=<value>] [--batch-size=<value>] [--pid-file-path=<value>] <consumer_name>
+bin/magento queue:consumers:start <consumer_name> [--max-messages=<value>] [--batch-size=<value>] [--pid-file-path=<value>] [--area-code=<value>]
```
[the remark this answers is not in the document]
Comment (gelanivishal) [reply _includes/config/message-queue-consumers.md]: I have updated it.

**Comment by gelanivishal** [comment]
@keharper I have committed as requested changes.

**Comment by keharper** [comment]
Running tests

**Comment by keharper** [comment]
Thanks @gelanivishal

**Comment by dshevtsov** [comment]
@keharper please add version labels

**Pull request merged**